cmd/policy.go

```diff
@@ -59,6 +59,38 @@ Operates directly on github and creates PRs for protected branches. Requires an
 	},
 }
 
+var terraformSubCmd = &cobra.Command{
+	// render all terraform templates into specific single repo
+	// then you can manually push the render PRs if is fine for you
+	Use:     "terraform",
```

Comment:
Suggest calling this `gpac` (github policy as code) instead of `terraform`. Please do suggest alternatives if you don't like my suggestion.

Reply:
Done